src/schematics/src/add/index.ts

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import {normalize, Path} from "@angular-devkit/core";
+import {chain, Rule, SchematicContext, SchematicsException, Tree} from "@angular-devkit/schematics";
+import {NodePackageInstallTask} from "@angular-devkit/schematics/tasks";
+import * as fs from "fs";
+import {join} from "path";
+import * as ts from "typescript";
+
+import {addSymbolToNgModuleMetadata} from "../utility/ast-utils";
+import {InsertChange} from "../utility/change";
+
+import {Schema as ComponentOptions} from "./schema";
+
+// Determine where to load the package.json, if doing local dev or not
+let corePackage: any;
+if (fs.existsSync(join(__dirname, "../../package.json"))) {
+    corePackage = require("../../package.json")
```

Comment:
Missing `;`

Reply:
Fixed

Reply:
Though we aren't tslinting this yet (because so much is copied over from Angular's utilities)

Reply:
I only saw it b/c InteliiJ had the red line in the changed file.